src/ui/game_window.c

```diff
@@ -42,6 +49,26 @@ void startGameLoop()
     freeGhostList();
 }
 
+// This function exists to handle the game events while the game is paused
+// Returns void, not really sure why handleGameEvents(); returns bool
+void handlePauseEvents()
+{
+    SDL_Event event;
+
+    while (!pGameQuit && SDL_PollEvent(&event))
+    {
+        switch (event.type)
+        {
+        case SDL_QUIT:
+            pGameQuit = true;
+            break;
+        }
+    }
+
+    if (keys[SDL_SCANCODE_P])
+        gamePause = !gamePause;
+}
+
 // This function should trigger all required events handling
 bool handleGameEvents()
```

Comment:
You have right, we don't need this function to return a boolean for now !
```suggestion
void handleGameEvents()
```

We also need to change this signature in `game_window.h`